pkg/bpf/bpf.go

```diff
@@ -30,6 +30,7 @@ import (
 
 	"kmesh.net/kmesh/daemon/options"
 	"kmesh.net/kmesh/pkg/logger"
+	pkgversion "kmesh.net/kmesh/pkg/version"
```

Comment:
Why not use version directly?
```suggestion
	"kmesh.net/kmesh/pkg/version"
```

Reply:
done